<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="textoRotuno">
         <property name="text">
          <string>Texto</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineaEditor"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="sincRotulo">
         <property name="text">
          <string>Sinc</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineaSinc">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>GroupBox</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QComboBox" name="comboEcho"/>
          </item>
          <item>
           <widget class="QCheckBox" name="marcaPermitir">
            <property name="text">
             <string>Permitir escribe</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="marcaSoloLeer">
            <property name="text">
             <string>Solo Leer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="marcaLimpiar">
            <property name="text">
             <string>Limpiar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnCometer">
     <property name="text">
      <string>Cometer</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
